test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
Gotcha - I read the docs meanwhile :) Yeah you'll need to configure the callbacks separately then, similarly to the client verify callback.